learning-examples/pumpswap/manual_buy_pumpswap.py

```
@@ -0,0 +1,323 @@
+    offset = 43  # Offset where the base_mint field is stored in the account data structure
+    filters = [
+        MemcmpOpts(offset=offset, bytes=base_mint_bytes)
+    ]
+        
+    response = await client.get_program_accounts(
+        amm_program_id,
+        encoding="base64",
+        filters=filters
+    )
+
+    market_address = [account.pubkey for account in response.value][0]
```

Ruff (0.8.2)

Prefer `next(account.pubkey for account in response.value)` over single element slice

Replace with `next(account.pubkey for account in response.value)`

(RUF015)